Changes.md

```diff
@@ -1,3 +1,11 @@
+0.59.x.x (relative to 0.59.9.1)
+========
+
+Features
+-----
+
+- Arnold Backend : Added visibility options for autobumps.
```

Comment:
The intended audience for Changes.md is the end user, so "ArnoldAttributes" would be a better category than "Arnold Backend", as its the place they will interact with the feature.